### DISCUSSION TITLE
Data Provider: Filter for Unique Row Id possible?

### DISCUSSION THREAD

**Discussion by Doctor-N0** [Budibase/budibase#11126] [Help] [answered]
That might be a very simple question, still I can't figure it out.

- filter a table in my Data Provider
- If there are two or more rows with the same `id`, only take the most recent (`updated_at` column of the table)

The result should be a filtered table with only "unique" ids.
 
Any ideas how this is possible in Budibase?

Thanks so much! 

PS: I've also posted this issue on [Stackoverflow](https://stackoverflow.com/questions/76611608/custom-filter-of-tables-in-budibase).

**Comment by ducnhoxlatui** [comment]
hmmm, i think in Budibase still having a _id which will define all the rows and no rows will have the same _id. So maybe you can see this as an example: 
![image](https://github.com/Budibase/budibase/assets/71976420/99b10416-be7e-4496-88a5-139369207b40)

**Reply by Doctor-N0** [reply]
Thank you for the swift answer. Could you maybe explain what a "ticket-state" is? Does this refer to a state of, e.g. a Button or a Table? Thanks so much!

**Reply by ducnhoxlatui** [reply, marked as answer]
yah, it is state in Budibase. Folow this link: https://docs.budibase.com/docs/app-state
